ivy_tests/test_ivy/test_frontends/test_numpy/test_ufunc/test_methods.py

```diff
@@ -0,0 +1,38 @@
+# global
+
+# local
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_method
+
+
+# at
+@handle_frontend_method(
+    init_name="at",
```

Comment:
please add the following parameters:
`class_tree=CLASS_TREE`

--

and update `init_name` to `method_name`

--

and remove `method_tree`

Reply:
dtype and value generation should be placed in here so that `dtype_and_x` is made available as argument below. currently `dtype_and_x` is not declared yet